### PR TITLE
Update Italian localization

### PR DESCRIPTION
**Pull request by M0Rf30**
As in the title,
Regards

### CODE REVIEW

File: Translations/it.po [hunk 1]
```diff
@@ -416,101 +416,158 @@ msgstr "Gioca da solo senza esporti sulla rete."
...
+msgid "Hellfire Shareware"
+msgstr "Demo Hellfire"
+
+#: Source/DiabloUI/selgame.cpp:94
+msgid "The host is running a different game than you."
+msgstr "L'host sta eseguendo un gioco diverso dal tuo."
+
+#: Source/DiabloUI/selgame.cpp:96
+msgid "The host is running a different game mode ({:s}) than you."
+msgstr "L'host sta eseguendo una modalità di gioco {{:s}) diversa dalla tua."
```
Comment (ephphatha) [inline Translations/it.po:442]:
```suggestion
msgstr "L'host sta eseguendo una modalità di gioco ({:s}) diversa dalla tua."
```

File: Translations/it.po [hunk 2]
```diff
@@ -11505,61 +11500,120 @@ msgstr "Celia"
...
+
+#, c-format
+#~ msgid "%i gold"
+#~ msgstr "%i monete"
+
+#~ msgid "Experience: "
+#~ msgstr "Esperienza: "
+
+#~ msgid "Next Level: "
+#~ msgstr "Livello Successivo: "
```
Comment (AJenbo) [inline Translations/it.po:11619]: This was not intentionally removed, I have fixed it in master now. If possible could you update the translation to still include these lines :)

**Review by AJenbo — APPROVED** [review]
Thank :)

**Comment by AJenbo** [comment]
@M0Rf30 there are 12 lines ready to be translated, we are preparing to release soon and hope to not change to much text before then :)

**Comment by M0Rf30** [comment]
Those lines will be in place before the release ;)

**Pull request merged**